Sources/Hummingbird/Middleware/LogRequestMiddleware.swift

```diff
@@ -12,32 +12,103 @@
 //
 //===----------------------------------------------------------------------===//
 
+import HTTPTypes
 import Logging
 
 /// Middleware outputting to log for every call to server
 public struct LogRequestsMiddleware<Context: BaseRequestContext>: RouterMiddleware {
+    /// Header filter
+    public enum HeaderFilter: Sendable, ExpressibleByArrayLiteral {
```

Comment:
What are the odds we'd be adding more cases after this? If the odds are we will add more cases, let's consider making this a `struct` enclosing an `internal enum`

Reply:
Yeah its always worthwhile